openpype/hosts/fusion/deploy/MenuScripts/install_pyside2.py

```diff
@@ -13,14 +25,17 @@
     print(f"Qt path: {mod.__file__}")
     print("Qt library found, nothing to do..")
 
-except ImportError:
+except Exception:
+    python_path = get_python_path()
+    if python_path is None:
+        print("Can't find any any python environment.")
```

Comment:
Can't we just rely on `sys.executable` at this stage since we're already running from a Python process?

Reply:
Had a quick look:
```python
import sys
print(sys.executable)
# C:\Program Files\Blackmagic Design\Fusion 18\Fusion.exe
```
So I guess that doesn't work.

However,
```
Py3> print(sys.base_prefix)
C:\Users\User\AppData\Local\Programs\Python\Python39
Py3> print(sys.base_exec_prefix)
C:\Users\User\AppData\Local\Programs\Python\Python39
Py3> print(sys.exec_prefix)
C:\Users\User\AppData\Local\Programs\Python\Python39
Py3> print(sys.prefix)
C:\Users\User\AppData\Local\Programs\Python\Python39
```
See also relevant documentation entries:
- https://docs.python.org/3/library/sys.html#sys.base_prefix
- https://docs.python.org/3/library/sys.html#sys.base_exec_prefix
- https://docs.python.org/3/library/sys.html#sys.exec_prefix
- https://docs.python.org/3/library/sys.html#sys.prefix

Could we rely on any of those?